app/controllers/api/host_initiators_controller.rb

```diff
@@ -9,5 +9,11 @@ def create_resource(type, _id = nil, data = {})
         {:task_id => klass.create_host_initiator_queue(User.current_userid, ems, data)}
       end
     end
+
+    def delete_resource_action(type, id = nil, _data = nil)
+      api_resource(type, id, "Detaching", :supports => :delete) do |host_initiator|
```

Comment:
Should this say `"Deleting"` instead of `"Detaching"`?

Reply:
yea, change this text and we can merge